sotodlib/coords/helpers.py

```diff
@@ -220,8 +220,10 @@ def get_wcs_kernel(proj, ra=None, dec=None, res=None):
     This interface is subject to change.
 
     Args:
-      proj (str): Name and resolution of projection to be used. It
-                  must adhere to the following format:
+      proj (str): Either the name of a FITS projection to use (e.g. 'car',
+                  'cea', 'tan'), in which case "res" must also be specified,
+                  or a string containing both the projection and the
+                  resolution, in this format:
```

Comment:
This formatting is technically acceptable but it is the convention in this file to indent the description only by 2 spaces.  i.e.
```
    Args:
      proj (str): Either the name of a FITS projection to use (e.g. 'car',
        'cea', 'tan'), in which case "res" must also be specified,
```